extensions-compose/src/commonMain/kotlin/com/arkivanov/decompose/extensions/compose/stack/animation/EmptyStackAnimation.kt

```diff
@@ -2,11 +2,24 @@ package com.arkivanov.decompose.extensions.compose.stack.animation
 
 import androidx.compose.foundation.layout.Box
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.compositionLocalOf
 import androidx.compose.ui.Modifier
 import com.arkivanov.decompose.Child
 import com.arkivanov.decompose.router.stack.ChildStack
 
-internal fun <C : Any, T : Any> emptyStackAnimation(): StackAnimation<C, T> =
+
+val LocalStackAnimation = compositionLocalOf<DefaultStackAnimationProvider> {
+    object : DefaultStackAnimationProvider {
+        override fun <C : Any, T : Any> invoke(): StackAnimation<C, T> =
+            emptyStackAnimation()
+    }
+}
+
+interface DefaultStackAnimationProvider {
+    operator fun <C : Any, T : Any> invoke(): StackAnimation<C, T>
```

Comment:
Let's make this a regular function named `provide`. Also please make the return type nullable.